zebra-rpc/src/methods.rs

```diff
@@ -1538,6 +1550,68 @@ impl Default for GetBlockChainInfo {
     }
 }
 
+impl GetBlockChainInfo {
+    /// Creates a new [`GetBlockChainInfo`] instance.
+    pub fn new(
+        chain: String,
+        blocks: Height,
+        best_block_hash: block::Hash,
+        estimated_height: Height,
+        value_pools: [types::ValuePoolBalance; 5],
+        upgrades: IndexMap<ConsensusBranchIdHex, NetworkUpgradeInfo>,
+        consensus: TipConsensusBranch,
+    ) -> Self {
+        Self {
+            chain,
+            blocks,
+            best_block_hash,
+            estimated_height,
+            value_pools,
+            upgrades,
+            consensus,
+        }
+    }
+
+    /// Returns the current network name as defined in BIP70 (main, test, regtest).
+    pub fn chain(&self) -> String {
+        self.chain.clone()
+    }
+
+    /// Returns the current number of blocks processed in the server.
+    pub fn blocks(&self) -> Height {
+        self.blocks
+    }
+
+    /// Returns the hash of the currently best block, in big-endian order, hex-encoded.
```

Comment:
Nitpick/Optional: This might be clearer here and in the `GetBlockChainInfo` field docs.

```suggestion
    /// Returns the hash of the current best chain tip block, in big-endian order, hex-encoded.
```

Reply:
Change made in [a65f847].